client/flutter/test/home_page_test.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:WHOFlutter/constants.dart';
+import 'package:WHOFlutter/generated/l10n.dart';
+import 'package:WHOFlutter/home_page.dart';
+
+void main() {
+  Widget testableWidget({Widget child}) {
```

Comment:
That's really good that you have extracted this functionality into a method instead of copying and pasting this in every test 👍